gulpfile.js

```diff
@@ -34,7 +34,11 @@ let manifest = {
 gulp.task('pug', () => {
 	const tags = Array.from(swagList.reduce(
 		(tagList, {tags}) => {
-			tags.forEach(tag => tagList.add(tag));
+			tags.forEach(tag => {
```

Comment:
```suggestion
 			tags.filter(tag => tag !== 'expired').forEach(tag => tagList.add(tag))
```

I'm not entirely sure how to make suggested changes for multiple lines